### PR TITLE
Update docs for counterfeiter

### PR DESCRIPTION
**Pull request by carolynvs**
* Show copy/pastable examples
* Mention that you _have_ to fix up the copyright header
* Fix command to install counterfeiter

### CODE REVIEW

File: docs/devguide.md [hunk 1]
```diff
@@ -313,9 +313,9 @@ and to diff the changes in the golden file to ensure that the new output is corr
 ### Counterfeiter
 Certain tests use fakes generated with [Counterfeiter](http://github.com/maxbrunsfeld/counterfeiter). If you add a method
 to an interface (such as SvcatClient in pkg/svcat/service-catalog) you may need to regenerate the fake. You can install
-Counterfeiter by running `go get http://github.com/maxbrunsfeld/counterfeiter && go install http://github.com/maxbrunsfeld/countefeiter`.
-Then regenerate the fake with `counterfeiter ./path/to/dir/with/interface InterfaceName`. You may have to manually paste the boilerplate
-comment into the fakes.
+Counterfeiter by running `go get github.com/maxbrunsfeld/counterfeiter`.
```
Comment (MHBauer) [inline docs/devguide.md]: Should we suggest master or a specific version? There is an aliased go-get site https://gopkg.in/maxbrunsfeld/counterfeiter.v5

Comment (carolynvs) [reply docs/devguide.md]: @jberkhahn Does it matter which version or is tip of master ok?

**Comment by jboyd01** [comment]
looks fine to me but awaiting @jberkhahn answer to https://github.com/kubernetes-incubator/service-catalog/pull/2228#discussion_r204527858

File: docs/devguide.md (hunk 1 above)
Comment (jberkhahn) [reply docs/devguide.md]: I'd be fine with pulling master.

**Comment by jboyd01** [comment]
I added my LGTM2 label, I see @jberkhahn did a slash LGTM, so adding a LGTM1 label for him as well.

**Pull request merged**